website_sale_stock_available_display/models/res_config_settings.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    inventory_availability = fields.Selection(selection_add=[
```

Comment:
Isn't this a related field and thus automatically populated the new value?

Reply:
I am extending this field
https://github.com/odoo/odoo/blob/87d7adaf16a0adc37c0772a28e6e736c5f0898e4/addons/website_sale_stock/models/res_config_settings.py#L10

Reply:
Oh, I would expect that Odoo would put there a related field for not needing to define 2 times the same selection. Very ugly...